src/elements/Contact/Contact.tsx

```diff
@@ -1,17 +1,16 @@
 import type { ContactNode } from '@prezly/story-content-format';
 import classNames from 'classnames';
-import type { FunctionComponent, HTMLAttributes } from 'react';
+import type { HTMLAttributes } from 'react';
 
 import { Avatar } from './Avatar';
 import './Contact.scss';
 import { SocialFields } from './SocialFields';
 
-interface Props extends HTMLAttributes<HTMLDivElement> {
-    children?: never;
+interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'children'> {
     node: ContactNode;
 }
 
-export const Contact: FunctionComponent<Props> = ({ children, className, node, ...props }) => {
```

Comment:
> children missing from component prop type on the Component component. Since React 18, FunctionComponent type no longer adds children property automatically. I got rid of the type entirely and updated the components to use named functions instead, to match our code-style in other repos.

I wonder why our automated tests did not catch this. We do test against React 16, 17, and 18:

https://github.com/prezly/content-renderer-react-js/actions/runs/4690828148/jobs/8314464680

Reply:
I guess the issue only comes up when building a TS project with Content Renderer as a dependency.